v3p/zlib/zconf.h

```diff
@@ -151,7 +145,7 @@
 #    define FAR far
 #  endif
 #endif
-#if defined(__BORLANDC__) && (defined(__SMALL__) || defined(__MEDIUM__))
+#if defined(__MEDIUM__))
```

Comment:
Sorry, I know ... I reviewed it a few other times too :(  I'll wait for
merging.

On Thu, Jan 14, 2016 at 7:20 AM, Hans Johnson <notifications@github.com>
wrote:

> In v3p/zlib/zconf.h
> <https://github.com/vxl/vxl/pull/105#discussion_r49719355>:
>
> > @@ -151,7 +145,7 @@
> >  #    define FAR far
> >  #  endif
> >  #endif
> > -#if defined(__BORLANDC__) && (defined(__SMALL__) || defined(__MEDIUM__))
> > +#if defined(__MEDIUM__))
>
> Arrg. I've removed that code at least 2 times already. I must have put the
> fix in a different patch set. I'll remove here again.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vxl/vxl/pull/105/files#r49719355>.
>



-- 
David Stoup
R&D Engineer

Kitware, Inc.
28 Corporate Drive
Clifton Park, NY. 12065
518-881-4949 (W)
518-312-3946 (M)
518-371-4573 (F)